oauth2client/crypt.py

```diff
@@ -95,7 +97,108 @@ def make_signed_jwt(signer, payload):
     return b'.'.join(segments)
 
 
-def verify_signed_jwt_with_certs(jwt, certs, audience):
+def _verify_signature(message, signature, certs):
```

Comment:
There is no really good reason to use a dictionary here, but that is the "native" format that the certs are given by Google: https://www.googleapis.com/oauth2/v1/certs

This URI is [specified][1] in `client.py`.

[1]: https://github.com/google/oauth2client/blob/5dda03224fbb408fbb3ac450733b997b525496c6/oauth2client/client.py#L67